### DISCUSSION TITLE
Wrong focus color

### DISCUSSION THREAD

**Discussion by florian-lefebvre** [supabase/supabase#870] [Questions] [answered]
Hi, this is a detail but on the index page, the hamburger button in the header has the wrong class on focus `focus:ring-indigo-500` and should be `focus:ring-brand-700` I think.
https://github.com/supabase/supabase/blob/a36b3ab4ad6f63bf1e41ae1fdffc58157f79a655/www/components/Nav/index.tsx#L111

**Comment by kiwicopple** [comment, marked as answer]
You're right! just pushed your updates. Thanks @florian-lefebvre 🙏

**Reply by florian-lefebvre** [reply]
@kiwicopple Same thing here 😉 (it's the last one on the index page):
https://github.com/supabase/supabase/blob/a36b3ab4ad6f63bf1e41ae1fdffc58157f79a655/www/components/MadeForDevelopers/CodeExamples.tsx#L108

**Reply by florian-lefebvre** [reply]
Also, I was wondering where the app.supabase.io custom scollbar style was in the repo, I couldn't find it. I think it would be great if it was also applied to supabase.io instead of the default one.

**Reply by florian-lefebvre** [reply]
Well, I'm going to do a PR because I have some other small corrections in mind as well.
